app/controllers/api/v1/orders_controller.rb

```diff
@@ -130,6 +130,7 @@ def order_record
         if order_params[:stockit_id]
           @order = Order.accessible_by(current_ability).where(stockit_id: order_params[:stockit_id]).first_or_initialize
           @order.assign_attributes(order_params)
+          @order.assign_state(params[:status])
           @order.stockit_activity = stockit_activity
           @order.stockit_contact = stockit_contact
           @order.stockit_organisation = stockit_organisation
```

Comment:
we need to change this code to accept stockit shipments as well as local orders... I wonder if we wait until after we have shutdown stockit then we don't need to do anything